src/main/scala/day5/Calculator.scala

```diff
@@ -0,0 +1,39 @@
+package day5
+
+class Calculator {
+  def sum(inputIntOne: Int, inputIntTwo: Int): Any = {
+    val inputIntOneLong = inputIntOne.toLong
+    val inputIntTwoLong = inputIntTwo.toLong
+    inputIntOneLong + inputIntTwoLong match {
```

Comment:
why declare `val inputIntOneLong = inputIntOne.toLong
    val inputIntTwoLong = inputIntTwo.toLong`

when you could just do 

`inputIntOne.toLong + inputIntTwo.toLong match`